common/changes/@microsoft/rush/dmichon-add-ignore-hooks_2020-10-14-23-17.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add --ignore-hooks flag",
```

Comment:
```suggestion
      "comment": "Add a --ignore-hooks flag to the rush install and rush update commands.",
```

Reply:
`"comment": "Add an --ignore-hooks flag to the \"rush install\" and \"rush update\" commands."`

Reply:
The underlying text has been updated to match the behavior following previous feedback to move the logic to the EventHooksManager and thereby to all rush actions